src/Dialogs/Plane/PlanePolarDialog.cpp

```diff
@@ -39,14 +39,16 @@ Copyright_License {
 #include "Language/Language.hpp"
 #include "UIGlobals.hpp"
 
+#include "cstring"
```

Comment:
Why this include?
External headers should be included with `<cstring>` not `"cstring"`.

Reply:
removed